sql/driver/wire.proto

```diff
@@ -40,6 +49,10 @@ message Datum {
     double float_val = 3;
     bytes bytes_val = 4;
     string string_val = 5;
+    Timestamp time_val = 6;
+    // TODO(vivek): Add additional types like Date and Interval that are
```

Comment:
But we can't implement the `ValueConverter` interface for a type in a different package like `time.Duration`. That type has to implement the interface itself. Perhaps I'm missing something. Can you expand on how you're thinking `ValueConverter` will help?